Update connect-jupyter-notebooks.md (#2568)
spotinst/help: src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md

 }
```

> **Warning**: add external jar dependencies to Scala Notebook:  
> - `deps.jars` field on the Configuration template does not work with Scala Notebook and **should not be set**
> - Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):
>   - Add a JAR with URL: `%AddJar <URL>`
>       - example: `%AddJar https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.20/postgresql-42.2.20.jar`
>   - Add a dependency from maven repo: `%AddDeps <group-id> <artifact-id> <version>`
>     - example: `%AddDeps org.postgresql postgresql 42.2.20`
>     - if the dependency has transitive dependencies, you can add the `--transitive` flag to add those dependencies  
> 
> More documentation for these [magic commands](https://toree.incubator.apache.org/docs/current/user/faq/): 
> 

Reviewer: ```suggestion
**Warning**: Adding external jar dependencies to Scala Notebooks  
The `deps.jars` field in the application configuration does not work with Scala Notebooks and **should not be set**. The JARs specified in this field will not be available on the driver Java classpath.

Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):
- Add a JAR with URL: `%AddJar <URL>`
```
%AddJar https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.20/postgresql-42.2.20.jar
```
- Add a dependency from maven repo: `%AddDeps <group-id> <artifact-id> <version>`
```
%AddDeps org.postgresql postgresql 42.2.20
```
If the dependency has transitive dependencies, you can add the `--transitive` flag to add those dependencies.  

More documentation for these magic commands is available in the [Toree documentation](https://toree.incubator.apache.org/docs/current/user/faq/)

```

Reviewer: GitHub messes up my comment, I'll send it to you on Slack

 }
```

> **Warning**: Adding external jar dependencies to Scala Notebooks  

Reviewer: ```suggestion
**Warning**: Adding external JAR dependencies to Scala Notebooks  
```

Author: 👍

The `deps.jars` field in the application configuration does not work with Scala Notebooks and **should not be set**. The JARs specified in this field will not be available on the driver Java classpath.

> Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):

Reviewer: ```suggestion
Instead, you can add an external JAR to the Spark context from the notebook itself with these magic commands (once the spark session is up):
```
